run

```diff
@@ -8,12 +8,18 @@ USAGE
   ./run [COMMAND] [options]
 
 COMMAND
-  server      run development server locally, access at 127.0.0.1:8080
-  db_shell    open a psql prompt using the provided DB credentials
-  db_tunnel   open an SSH tunnel through music.volfied.net to the RDS instance
-  lint        run black and isort
-  tests       run the full test suite
-  tests_fast  run only the quicker tests, excluding db and integration tests
+  server        run development server locally, access at 127.0.0.1:8080
+  db_shell      open a psql prompt using the provided DB credentials
+  db_tunnel     open an SSH tunnel through music.volfied.net to the RDS instance
```

Comment:
Should music.volfied.net really be hardcoded here? Seems like it ought to be an environment variable and it ought to be to prod, anyway, so digimusings or whatever.

Reply:
We'll handle this in config.